### PR TITLE
Shared wordlist for frontend

### PR DESCRIPTION
**Pull request by maciejhirsz**
Analog to #5331 - use external repo with single source for the wordlist recovery phrases are generated from.

### CODE REVIEW

File: js/webpack/app.js [hunk 1]
```diff
@@ -67,7 +67,7 @@ module.exports = {
       },
       {
         test: /\.js$/,
-        include: /node_modules\/(material-chip-input|ethereumjs-tx)/,
```
Comment (tomusdrw) [inline js/webpack/app.js]: Is this necessary? I think the code should work in any browser supporting ES6. Do we support other browsers anyway?

Comment (maciejhirsz) [inline js/webpack/app.js]: It shouldn't be. We are still using UglifyJS to pack the bundle, and UglifyJS only understands ES5 syntax. I think the only thing that is ES6 there are `const` and `let` declarations, so we could either kick those out, or change the minifier to Babili or something else.

Comment (tomusdrw) [inline js/webpack/app.js]: Ah, I see. So it's ok then, was just making sure that it's needed.

Comment (jacogr) [inline js/webpack/app.js]: I have something in the back of my head to look at the new Babel uglifier, thanks for the reminder in logging that - just need to see if there are any benefits/drawbacks.

**Pull request merged**